src/LocalizeContext.js

```diff
@@ -77,17 +77,9 @@ export const getContextPropsFromState = (
       const defaultLanguage =
         options.defaultLanguage || (languages[0] && languages[0].code);
       const renderToStaticMarkup = options.renderToStaticMarkup;
-
-      /* @Note: Flow keeps complaining that the possibly-undefined
-       * options.ignoreTranslateChildren cannot be assigned to a boolean
-       * property, despite various attempts to narrow it to boolean.
-       */
-      let ignoreOption = {
-        ignoreTranslateChildren: defaultTranslateOptions.ignoreTranslateChildren
-      };
-      if (options.ignoreTranslateChildren != null) {
-        ignoreOption.ignoreTranslateChildren = options.ignoreTranslateChildren;
-      }
+      const ignoreTranslateChildren =
+        options.ignoreTranslateChildren ||
+        defaultTranslateOptions.ignoreTranslateChildren;
```

Comment:
I may be overthinking this one, but if one day it is decided that the default `ignoreTranslateChildren` is `true`, it will override a potential `false` being sent in by the user. Since the default is determined by the library itself probably the best thing is to add a test to help current/future maintainers.

Reply:
Actually it looks like this will be tricky to test due to the Enzyme problem with Context... It seems like it's not possible to trigger the whole flow of:
1. Instantiating the Provider (with default `ignoreTranslateChildren: true`)
2. Calling `initialize()` with `ignoreTranslateChildren: false`
3. Instantiating `Translate` and checking that the child was rendered.

Reply:
Yea I think this can be updated to evaluate similar to how I handled ignoreTranslateChildren in the Translate component, which should handle this scenario.